pkg/storage/mongo/saver.go

```diff
@@ -1,9 +1,16 @@
 package mongo
 
-import "github.com/gomods/athens/pkg/storage"
+import (
+	"github.com/gomods/athens/pkg/storage"
+
+	"github.com/gobuffalo/buffalo"
+)
+
+// asserts that a ModuleStore is a storage.Saver
+var _ storage.Saver = &ModuleStore{}
```

Comment:
same here RE not needing this. there are a few below too

Reply:
Fixed locally